ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -820,7 +821,32 @@ def modify_sssd_conf(host, domain, mod_dict, provider='ipa',
         with open(temp_config_file, 'wb') as f:
             f.write(current_config)
 
-        sssd_config = SSSDConfig()
+        # In order to use SSSDConfig() locally we need to import the schema
+        # Create a tar file with /usr/share/sssd.api.conf and
+        # /usr/share/sssd/sssd.api.d
+        cmd = host.run_command(['mktemp'])
+        tmpname = cmd.stdout_text.strip()
+        cmd = host.run_command(
+            ['tar', 'cJvf', tmpname, '--ignore-failed-read',
+             'sssd.api.conf',
+             'sssd.api.d'],
+            log_stdout=False, raiseonerr=False, cwd="/usr/share/sssd")
+        # fetch tar file
+        tar_dir = tempfile.mkdtemp()
+        tarname = os.path.join(tar_dir, "sssd_schema.tar.xz")
+        with open(tarname, 'wb') as f:
+            f.write(host.get_file_contents(tmpname))
+        # delete from remote
+        host.run_command(['rm', '-f', tmpname])
+        # Unpack on the local side
+        ipautil.run([paths.TAR, 'xJvf', tarname], cwd=tar_dir,
+                    raiseonerr=False)
```

Comment:
Why do you ignore errors here? I think we want to detect problems immediately when they appear.

Reply:
Good catch, I blindly copied/pasted other code